packages/data-store/src/utils/contact/MappingUtils.ts

```diff
@@ -202,27 +224,38 @@ export const physicalAddressFrom = (physicalAddress: PhysicalAddressEntity): Phy
     provinceName: physicalAddress.provinceName,
     countryCode: physicalAddress.countryCode,
     buildingName: physicalAddress.buildingName,
+    ownerId: physicalAddress.ownerId,
+    tenantId: physicalAddress.tenantId,
     createdAt: physicalAddress.createdAt,
     lastUpdatedAt: physicalAddress.lastUpdatedAt,
   }
 }
 
-export const identityEntityFrom = (args: NonPersistedIdentity): IdentityEntity => {
+export const identityEntityFrom = (entity: NonPersistedIdentity): IdentityEntity => {
   const identityEntity: IdentityEntity = new IdentityEntity()
-  identityEntity.alias = args.alias
-  identityEntity.roles = args.roles
-  identityEntity.identifier = correlationIdentifierEntityFrom(args.identifier)
-  identityEntity.connection = args.connection ? connectionEntityFrom(args.connection) : undefined
-  identityEntity.metadata = args.metadata ? args.metadata.map((item: NonPersistedMetadataItem) => metadataItemEntityFrom(item)) : []
-
+  identityEntity.alias = entity.alias
+  identityEntity.origin = entity.origin ?? IdentityOrigin.EXTERNAL
```

Comment:
just making a comment here to check the default value structure

Reply:
"?? IdentityOrigin.EXTERNAL" was in the branch when I took it over, but since it's mandatory in the entity, I've also made it mandatory in the DTO. It will break compatibility, but this is the way to bring to to the developer's attention.
This is the way...

Reply:
ok yes i meant the partyType. if i look at the code, we use different approaches to set default values for the partyType and the identity. and i wonder why